book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    rentals << rental
+  end
```

Comment:
- [ ] Instead of creating the rental and then passing it as a parameter here. Kindly pass the parameters so that you can use `self` to achieve the 'has-many' association.

```ruby
  def add_rental(person, date)
    Rental.new(date, self, person)
  end
```